examples/step-10/step-10.cc

```diff
@@ -118,7 +112,7 @@ namespace Step10
             // explicitly.
 
 
-            // In degree to actually write out the present grid with this
+            // In order to actually write out the present grid with this
```

Comment:
This substitution error is way funnier than it should be to me :)

Reply:
That mistake was added in 767a39c635: it took us nearly seventeen years to fix this typo in this program. Better late than never.